engines/agi/systemui.cpp

```diff
@@ -723,8 +743,13 @@ void SystemUI::drawSavedGameSlots() {
 	_text->charAttrib_Set(0, 15);
 
 	for (slotNr = 0; slotNr < slotsToDrawCount; slotNr++) {
-		_text->displayTextInsideWindow("-", 5 + slotNr, 1);
-		_text->displayTextInsideWindow(_savedGameArray[_savedGameUpmostSlotNr + slotNr].displayText, 5 + slotNr, 3);
+		if (!_vm->isLanguageRTL()) {
+			_text->displayTextInsideWindow("-", 5 + slotNr, 1);
+			_text->displayTextInsideWindow(_savedGameArray[_savedGameUpmostSlotNr + slotNr].displayText, 5 + slotNr, 3);
+		} else {
+			_text->displayTextInsideWindow(_savedGameArray[_savedGameUpmostSlotNr + slotNr].displayText, 5 + slotNr, 0);
+			_text->displayTextInsideWindow("-", 5 + slotNr, 32);
```

Comment:
What's 32? Please avoid magic numbers.

Reply:
Well, sometime there are numbers that are just magical...
The engine has a lot of such examples.

In this case, `32` is based on the magical `34` of 

	_text->drawMessageBox(slotListText, messageBoxHeight, 34, true);